ProjetSupaero2018/catkin_ws/src/display/scripts/spawn_all_cylinders.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import rospy
+import numpy as np
+import spawn_cylinder_client
+
+PARAM_NAME_SIZE = '/obstacles/obstacles_size'
+PARAM_NAME_OBSTACLES = '/obstacles/obstacles_vec'
+READ_OBSTACLES_SERVICE = 'read_obstacles'
+SPAWN_CYLINDER_SERVICE = 'spawn_cylinder'
+
+if __name__ == "__main__":
+    rospy.wait_for_service(READ_OBSTACLES_SERVICE)
+    rospy.wait_for_service(SPAWN_CYLINDER_SERVICE)
+
+    try:
+        vec = rospy.get_param(PARAM_NAME_OBSTACLES)
+        size = rospy.get_param(PARAM_NAME_SIZE)
+    except KeyError:
+        rospy.logerr('Parameter not set')
```

Comment:
If you have this exception, the remaining of the code will still run and fail. So maybe you would need a `return` statement after this line ?